src/view/components/overrides/overrides.jsx

```diff
@@ -9,187 +9,379 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
+              }
+              const itemIsInPropertyTokenOptions = createIsItemInArray(
+                propertyTokenOptions.map(({ value }) => value),
+                { errorOnEmptyArray: false, errorOnEmptyItem: false }
+              );
+              const isValidPropertyTokenOption = value =>
+                isDataElement(value) || itemIsInPropertyTokenOptions(value);
+
+              /** @type {string[]} */
+              const primaryReportSuites =
+                result?.com_adobe_analytics?.reportSuites ?? [];
+              const reportSuiteOptions =
+                primaryReportSuites
+                  .concat(result?.com_adobe_analytics?.reportSuites__additional)
+                  .filter(Boolean)
+                  .map(value => ({ value, label: value })) ?? [];
+              const itemIsInReportSuiteOptions = createIsItemInArray(
+                reportSuiteOptions.map(({ value }) => value),
+                { errorOnEmptyArray: false, errorOnEmptyItem: false }
+              );
+              const isValidReportSuiteOption = value => {
+                if (value?.includes(",")) {
+                  return value
+                    .split(",")
+                    .map(v => v.trim())
+                    .every(
+                      v => isDataElement(v) || itemIsInReportSuiteOptions(v)
+                    );
+                }
+                return (
+                  isDataElement(value) || itemIsInReportSuiteOptions(value)
+                );
+              };
+              const sandboxFieldName = `${prefix}.${env}.${
+                FIELD_NAMES.sandbox
+              }`;
+              const [{ value: sandbox }] = useField(sandboxFieldName);
+
+              return (
+                <Item key={env}>
+                  <Flex
+                    direction="column"
+                    marginX={largeHeader ? "" : "size-300"}
+                    gap="size-100"
+                  >
+                    <SettingsCopySection currentEnv={env} onPress={onCopy} />
+                    {!hideFieldsSet.has(FIELD_NAMES.datastreamId) && (
+                      <>
+                        <SandboxSelector
+                          data-test-id={FIELD_NAMES.sandbox}
+                          initInfo={initInfo}
+                          label="Sandbox"
+                          name={sandboxFieldName}
+                          width="size-5000"
+                        />
+                        <DatastreamSelector
+                          data-test-id={FIELD_NAMES.datastreamId}
+                          label="Datastream"
+                          description={`Override the default datastream${
+                            envEdgeConfigIds.datastreamId
+                              ? ` (${envEdgeConfigIds.datastreamId})`
+                              : ""
+                          }. Does not support cross-organiztion datastream overrides.`}
```

Comment:
"Override the configured datastream ${...}. This does not support cross-organization datastream overrides."